lib/ansible/plugins/callback/default.py

```diff
@@ -36,7 +36,8 @@
 COMPAT_OPTIONS = (('display_skipped_hosts', C.DISPLAY_SKIPPED_HOSTS),
                   ('display_ok_hosts', True),
                   ('show_custom_stats', C.SHOW_CUSTOM_STATS),
-                  ('display_failed_stderr', False),)
+                  ('display_failed_stderr', False),
+                  ('check_mode_markers', True),)
```

Comment:
why are you adding to compat options? this is new option

Reply:
Sorry about that. I don't know how to add the new type option. In any case I did what is done for `display_ok_hosts` which is a new type option. What should I do?